Source/WebCore/css/CSSPrimitiveValue.h

```diff
@@ -155,42 +157,66 @@ class CSSPrimitiveValue final : public CSSValue {
     ~CSSPrimitiveValue();
 
     CSSUnitType primitiveType() const;
-    ExceptionOr<float> getFloatValue(CSSUnitType) const;
-
-    double computeDegrees() const;
-    template<typename T, TimeUnit> T computeTime() const;
-    template<typename T, FrequencyUnit> T computeFrequency() const;
-
-    template<typename T> T computeLength(const CSSToLengthConversionData&) const;
-    template<int> Length convertToLength(const CSSToLengthConversionData&) const;
 
+    // Exposed for DeprecatedCSSOMPrimitiveValue. Throws if conversion to `targetUnit` is not allowed.
+    ExceptionOr<float> getFloatValueDeprecated(CSSUnitType targetUnit) const;
+
+    // MARK: Integer (requires `isInteger() == true`)
+    template<typename T = int> T resolveInteger(const CSSToLengthConversionData&) const;
```

Comment:
Not sure why this name doesn’t include `As`.

Reply:
Oops. Nope, just a goof. Fixed.